### ISSUE TITLE
6.2.1 Cache Server quits immediately without error

### ISSUE THREAD

**Issue by timezombi** [Unity-Technologies/unity-cache-server#86]
This might be a dupe of #84 but I thought I would mention this anyway.

Except for the OOM errors we were receiving v6.1.2 when making large file requests, it has been very stable.  After upgrading to 6.2.1, Unity Cache Server exits immediately without any errors or warnings.

With log level set to debugging, only this displays before crashing...

> Cache path is {OurCacheFilePath}

Node version is v8.14.0. NPM is 6.4.1.

**Comment by timezombi** [comment]
I can give you more information about the version of all of our packages and our config. Whatever you might need to help troubleshoot this!

**Comment by stephen-palmer** [comment]
Strange. So you're seeing a crash immediately without client connections?

What is your startup command line?

**Comment by timezombi** [comment]
That is correct.

The startup command looks like this (piped into another couple commands but it still fails when run alone):
`node --max-old-space-size=8192 /usr/local/bin/unity-cache-server -l 5 --NODE_CONFIG_DIR=/usr/local/bin/config`

I'm starting to wonder if including the log level flag with the NODE_CONFIG_DIR might be conflicting. When I originally put this in production, Global.logLevel didn't exist. I can create a test instance and mess around with the config, if that helps.

Inside /usr/local/bin/config is a default.yml file:
```
Cache:
  defaultModule: "cache_fs"
  options:
    processor:
      putWhitelist: []
    cache_ram:
      cachePath: "/Unity"
      pageSize: 100000000
      maxPageCount: 10
      minFreeBlockSize: 1024
      persistence: true
      persistenceOptions:
        autosave: true
        autosaveInterval: 10000
        throttledSaves: false
      highReliability: false
      highReliabilityOptions:
        reliabilityThreshold: 2
        saveUnreliableVersionArtifacts: true
        multiClient: false
    cache_fs:
      cachePath: "/Unity"
      cleanupOptions:
        expireTimeSpan: "P30D"
        maxCacheSize: 0
      persistence: true
      persistenceOptions:
        autosave: true
        autosaveInterval: 10000
        throttledSaves: false
      highReliability: false
      highReliabilityOptions:
        reliabilityThreshold: 2
        saveUnreliableVersionArtifacts: true
        multiClient: true
Mirror:
  options:
    queueProcessDelay: 2000
    connectionIdleTimeout: 10000
Server:
  options:
    allowIpv6: false
```

**Comment by stephen-palmer** [comment]
Fixed in release 6.2.3
